src/build-extensions-container.sh

```diff
@@ -34,8 +34,9 @@ img=localhost/extensions-container
 (set -x; podman build --from oci-archive:"$ostree_ociarchive" --network=host \
     --build-arg COSA=true --build-arg VARIANT="${variant}" --label version="$buildid" \
     --volume /etc/pki/ca-trust:/etc/pki/ca-trust:ro \
-    --volume "${workdir}"/tmp/extensions.json:/tmp/extensions.json \
     -t "${img}" -f extensions/Dockerfile "${ctx_dir}")
 
+(set -x; podman run --rm "${img}" cat /usr/share/rpm-ostree/extensions.json > "${workdir}/tmp/extensions.json")
```

Comment:
Worth noting ttis requires https://github.com/openshift/os/pull/1772 

random aside.. wish we had `podman image cp`: https://github.com/containers/podman/discussions/12658#discussioncomment-12528126

Reply:
Right yeah, this PR and the openshift/os one need to merge together.